sopel/coretasks.py

```diff
@@ -132,8 +190,8 @@ def enable_service_auth(bot, trigger):
             'owner.')
 
 
-@sopel.module.event(events.ERR_NOCHANMODES)
-@sopel.module.priority('high')
+@module.event(events.ERR_NOCHANMODES)
+@module.priority('high')
 def retry_join(bot, trigger):
     """Give NickServer enough time to identify on a +R channel.
```

Comment:
This annoys me, even though it's not part of this patch. (The typo is quite literally [6 years old](https://github.com/sopel-irc/sopel/commit/0540f998596d5e041e35fbedbba7d48bf669c46d).) I don't mind an unrelated tweak if you don't!

```suggestion
    """Give NickServ enough time to identify on a +R channel.
```